lib/BankAccount.php

```diff
@@ -89,7 +89,7 @@ public static function retrieve($_id, $_opts = null)
                "'bank_account_id')` or `Account::retrieveExternalAccount(" .
                "'account_id', 'bank_account_id')`.";
 
-        throw new Exception\BadMethodCallException($msg, null);
+        throw new Exception\BadMethodCallException($msg);
```

Comment:
The second argument to [`Exception.__construct`](https://www.php.net/manual/en/exception.construct.php) is optional, but when present should be an integer. `null` is not a valid value.